libs/gltfio/src/AssetLoader.cpp

```diff
@@ -1268,7 +1264,7 @@ AssetLoader* AssetLoader::create(const AssetConfiguration& config) {
 }
 
 void AssetLoader::destroy(AssetLoader** loader) {
```

Comment:
If we follow the Filament convention, `AssetLoader::destroy` should call `FAssetLoader::destroy` which would definitely prevent this kinds of mistakes.